docker/test/util/Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM ubuntu:20.04
 ARG apt_archive="http://archive.ubuntu.com"
 RUN sed -i "s|http://archive.ubuntu.com|$apt_archive|g" /etc/apt/sources.list
 
+# 15.0.2
```

Comment:
Does it mean - you simply force rebuilding the docker image?

Reply:
Yes